docs/src/api/class-browsertype.md

```diff
@@ -149,11 +149,19 @@ deleted when browser is closed.
 Enable Chromium sandboxing. Defaults to `false`.
 
 ### option: BrowserType.launch.firefoxUserPrefs
+* langs: js, python
 - `firefoxUserPrefs` <[Object]<[string], [string]|[float]|[boolean]>>
 
 Firefox user preferences. Learn more about the Firefox user preferences at
 [`about:config`](https://support.mozilla.org/en-US/kb/about-config-editor-firefox).
 
+### option: BrowserType.launch.firefoxUserPrefs
+* langs: csharp, java
+- `firefoxUserPrefs` <[Object]<[string], [string]>>
```

Comment:
Our browser_patches/firefox/preferences/playwright.cfg is full of boolean and number values.